scripts/vep/run_vep.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+
+"""
+Run VEP on the hail mt
+"""
+
+
+import click
+import hail as hl
+from cpg_utils.hail_batch import output_path
+
+
+@click.command()
+@click.option('--mt', required=True, help='Hail matrix table to run VEP on')
+@click.option('--vep-version', help='Version of VEP', default='104.3')
+def main(mt: str, vep_version: str):
+    """
+    Run vep using main.py wrapper
+    """
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(mt)
+    ht = mt.rows()
+    # filter to biallelic loci only
+    ht = ht.filter(hl.len(ht.alleles) == 2)
+    ht = ht.filter(ht.alleles[1] != '*')
```

Comment:
Can you add a comment explaining this filter (and maybe a link to the forum post), please?

Reply:
added